### DISCUSSION TITLE
What ever happened to the pytest injection branch?

### DISCUSSION THREAD

**Discussion by rtaycher** [adriangb/di#86] [Q&A]
Is there still plans to add it to this library?

**Comment by adriangb** [comment]
Just being busy and technical roadblocks happened 😅. I do think it would be really cool to have. Do you have any interest in working on it?

**Reply by rtaycher** [reply]
Sorry I don't think I can. At least not right now. I ended up making this small thing 
https://github.com/rtaycher/pytest-fixture-ref
which is a bit of a hack over current pytest system but probably good enough for me/for now

**Reply by adriangb** [reply]
Oh I’d seen that, I just didn’t put this discussion and that together!

I’m thinking it might make sense to just extend your package instead of directly using `di`. Performance, flexible executors, etc matter a lot less in tests. So maybe we can just extend your package a bit with some ideas from di?

**Reply by rtaycher** [reply]
don't know about that. My package is just a bit of a hack, to really do that I or someone would need to go a lot deeper into pytest.
For now the only thing I'm thinking about improving, is to use python-forge to replace the use of exec
